public/api-documentation/swagger.json

```diff
@@ -262,6 +262,46 @@
         }
       }
     },
+    "/manifests/{id}/cromerr_submit_transaction": {
```

Comment:
we generally try and keep verbs out of the URL. The verb is the HTTP method. The URL is the noun.